Common/Python/PandasConverter.cs

```diff
@@ -140,7 +140,7 @@ public PyObject GetIndicatorDataFrame(IDictionary<string, List<IndicatorDataPoin
                     pyDict.SetItem(kvp.Key.ToLowerInvariant(), _pandas.Series(data: values, index: index));
                 }
 
-                return _pandas.DataFrame.from_dict(pyDict);
+                return _pandas.DataFrame(pyDict, columns: data.Keys.Select(x => x.ToLowerInvariant()).OrderBy(x => x));
```

Comment:
I think that if the column names are not already sorted, this will give back the dataframe with the wrong column names.

Let's make a test to make sure that the column names are in the order we expect 👍

Reply:
That's exactly what I was thinking in the conversation with Martin above, but upon testing it, it actually matched the columns to the correct values in the dictionary. I'll go ahead and figure a way to add a test to confirm that behavior.